### ISSUE TITLE
frontend/ check if frontend complies with our contrast requirements 

### ISSUE THREAD

**Issue by ArdaYalcindag** [bounswe/bounswe2025group5#571]
### Task Description
The front should be checked it complies with our contrast requirement which is: "Minimum color contrast ratio of 4.5:1 for text and 3:1 for large text/UI shall be maintained.", as in the [SRS](https://github.com/bounswe/bounswe2025group5/wiki/Requirements-of-CMPE451#212-usability-and-accessibility-requirements).

### Acceptance Criteria
- [ ] Checked if the requirement had been already met.
- [ ] Implemented the part if it hadn't been implemented.

### Deadline
08.11.2025 - 20.00

**Comment by ArdaYalcindag** [comment]
I have checked the whole frontend. Most of the texts were complying our contrast ratio. Made the changes for the ones who weren't. Here is the [PR](https://github.com/bounswe/bounswe2025group5/pull/572). @abdarslan Can you review and close the issue and PR?
